docs/development/development.md

```diff
@@ -90,6 +96,9 @@ To locally run Akri's Agent, Controller, and Discovery Handlers as part of a Kub
     cd akri/agent
     sudo -E DEBUG_ECHO_INSTANCES_SHARED=true ENABLE_DEBUG_ECHO=1 RUST_LOG=info METRICS_PORT=8082 KUBECONFIG=$HOME/.kube/config DISCOVERY_HANDLERS_DIRECTORY=~/tmp/akri AGENT_NODE_NAME=myNode HOST_CRICTL_PATH=/usr/bin/crictl HOST_RUNTIME_ENDPOINT=/var/run/dockershim.sock HOST_IMAGE_ENDPOINT=/var/run/dockershim.sock $HOME/.cargo/bin/cargo run
     ```
+    > Note: `DISCOVERY_HANDLERS_DIRECTORY` is where Akri agent creates an unix domain socket for discovery handler's registeration. This example uses /tmp/akri that should exist or is created before executing this command.
```

Comment:
Can you add a new line before the note? I believe our CMS for docs needs the notes to be isolated with newlines. Also unnecessary extra newline seems to be added after.

Reply:
Thanks. I will format accordingly. Btw, I am not sure if it is my fault or there is other missing step in this chapter. Right now my locally running debug echo discovery handler does not really successfully register to also locally running Akri agent, I explain in the issue report. Appreciate much if you can help.